Scripts/installers.ps1

```diff
@@ -14,6 +14,7 @@ function Stage-Installer {
     }
 
 	Write-Log "Downloading installer '$downloadUrl' to '$MsiPath' ..."
+  [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Changing this to `[Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]::Tls12` enables TLS 1.2 alongside any existing options, which is preferable.

Reply:
Done!

Could you, please, check again?

Reply:
Looks good.